src/Reference.php

```diff
@@ -20,6 +20,14 @@ class Reference
     use \atk4\core\DIContainerTrait;
     use \atk4\core\FactoryTrait;
 
+    /**
+     * Owner Model of the reference.
+     * override the hint type definition already present in TrackableTrait.
+     *
+     * @var Model
+     */
+    public $owner;
```

Comment:
Defined in https://agile-core.readthedocs.io/en/develop/container.html?highlight=owner#trackable-trait

Reply:
i know but intellisense get defintion of owner model as object in place of Model and don't work.

Reference has always a ->owner = Model, no? why not type hint it and let intellisense autocomplete?

Reply:
fine with me if that helps IDEs